src/ui/controllers/user.ts

```diff
@@ -378,7 +378,9 @@ export async function renderEditPage(
 
 			break
 		case 'department':
-			response = await registry.getWithoutHal('/organisationalUnits/flat')
+			const email = req.user.userName
+			const domain = email.split("@")[1]
+			response = await registry.getWithoutHal('/organisationalUnits/flat/' + domain + '/')
```

Comment:
Above three lines of code change are redundant. Could you please revert them back.